src/Aspire.Hosting.Azure/Provisioning/Provisioners/BicepProvisioner.cs

```diff
@@ -198,48 +205,54 @@ await notificationService.PublishUpdateAsync(resource, state =>
 
         if (deployment.Data.Properties.ProvisioningState == ResourcesProvisioningState.Succeeded)
         {
-            template.Dispose();
+            if (context.ExecutionContext.IsRunMode)
```

Comment:
We want to keep the template files used to do the deployment around during deploy mode (for now: see https://github.com/dotnet/aspire/pull/10642).